docs/source/terms_and_model.rst

```diff
@@ -346,7 +346,119 @@ Location for Variation.
   location is non-specific (e.g., a range) or symbolic (a gene).
 
 
+.. _chromosome_location:
+
+ChromosomeLocation
+$$$$$$$$$$$$$$$$$$$$
+
+**Biological definition**
+
+Imprecise chromosomal locations based on named landmarks assayed
+through chromosomal staining, FISH, recombination studies, or other
+low-resolution methods.
+
+**Computational definition**
+
+A ChromosomeLocation is a :ref:`Location` that is defined by a named
+chromosomal band within a species.  Conceptually, a ChromosomeLocation
+represents a family of :ref:`SequenceLocation` on distinct
+:ref:`Sequence` entities.
```

Comment:
It took me a minute to understand what was meant by the family of SequenceLocatio on Sequence entites. I get it, but I wonder if it is useful to have this in the definition–I feel like it does not clarify the preceding sentence, and could cause confusion to people who do not already understand the concept being presented here. I would drop that part of the definition.
```suggestion
chromosomal band within a species.
```

Reply:
I agree. Too wordy.